exchanges/kraken.py

```diff
@@ -19,17 +39,63 @@ def latest_trade(self, asset_pair):
 
         ticker = self._public_query('Ticker', {'pair': [asset_pair]})
         price, volume = ticker[asset_pair]['c']
+
         return Trade(asset_pair, price, volume)
 
     def place_order(self, order):
         pass
 
     def order_info(self, order_id):
-        pass
+        """Queries information about an order"""
+
+        info = self._private_query('QueryOrders', {'txid': order_id})
+        order_info = OrderInformation(info[order_id]['status'])
+
+        return order_info
 
     def trades_history(self, asset_pair, start_time, finish_time):
         pass
 
+    def _private_query(self, query_name, data=None):
+        """Queries private data on Kraken"""
+
+        if self.public_key is None or self.private_key is None:
+            raise ValueError('Querying the private user '
+                             'data requires an API key.')
```

Comment:
And here I'll put "Please add your public '
                           'and private keys to the file '
                           '\'exchanges/data/private/kraken-key\'.'"